src/Microsoft.OpenApi.OData.Reader/Edm/EdmModelExtensions.cs

```diff
@@ -152,6 +152,7 @@ public static IEnumerable<IEdmComplexType> FindAllBaseTypes(this IEdmComplexType
         /// <param name="baseType">Type of the base type.</param>
         /// <param name="subtype">Type of the sub type.</param>
         /// <returns>true, if the <paramref name="baseType"/> is assignable to <paramref name="subtype"/>. Otherwise returns false.</returns>
+        [Obsolete]
```

Comment:
why is this method tagged as obsolete? since it's part of the public API, we should at least provide a rationale

Reply:
This shouldn't have been here, this was made obsolete in this other [PR 160](https://github.com/microsoft/OpenAPI.NET.OData/pull/160), from this [suggestion](https://www.youtube.com/watch?v=-0uPUgdTbnw). I have corrected it.